.circleci/config.yml

```diff
@@ -176,7 +211,11 @@ workflows:
             - unit_test
             - integration_test
             - build_docker
-
+      - run_smoke_test:
+          name: smoke-test-dev
+          env: dev
+          requires:
+            - deploy_dev
```

Comment:
Not currently stopping preprod approval so we can see how the smoke tests go - hopefully it will eventually replace preprod approval.